controllers/webspherelibertyapplication_controller.go

```diff
@@ -58,6 +58,16 @@ const (
 	OperatorName = "websphere-liberty-operator"
 )
 
+// Labels used for Operator resources (not for CRs)
+func GetOperatorLabels() map[string]string {
+	labels := map[string]string{
+		"app.kubernetes.io/instance":   OperatorName,
+		"app.kubernetes.io/name":       OperatorName,
+		"app.kubernetes.io/managed-by": "olm",
```

Comment:
Since the users can also set config values in the ConfigMap, I think we should set it to "user".

It'll be good to add the same for RCO and OLO to keep it consistent. So the labels can be directly set inside `CreateConfigMap` method and avoid passing in the second parameter.